app/models/shopify/import/base.rb

```diff
@@ -0,0 +1,45 @@
+module Shopify
+  module Import
+    class Base
+      def initialize(credentials: {}, client: nil)
```

Comment:
@Argonus I noticed you forgot to reply to this comment. Can you please check it out?

The change I propose would also benefit if we'd later switch to using `shopify_app` gem, as it will be easier to just pass the connection as needed. (or to establish it, as needed, elsewhere)